system/View/Table.php

```diff
@@ -83,6 +83,11 @@ class Table
      */
     public $function;
 
+    /**
+     * Order each inserted row by heading keys
+     */
+    public bool $syncRowsWithHeading = false;
```

Comment:
Is there a need to make this public?

Reply:
I think not.

Reply:
All other properties are `public`, I will keep it for consistency

Reply:
All other properties seems to be wrong, but changing them is breaking changes...

Reply:
@paulbalandan 
1. `public` for current consistency
2. change only this property to `protected` or `private`

Reply:
I would like this new property to be `private` to enforce API encapsulation. We can increase visibility later if needed.
For the others, we can retain them as `public` to prevent breaking changes.